dist/less/overlay-panel.less

```diff
@@ -89,6 +103,9 @@ body.overlay-open {
     height: @wizard-pf-steps-indicator-height;
     margin: @wizard-pf-steps-indicator-margin-top-bottom 0;
     padding: 0;
+    @media(max-width: @screen-sm-min) {
+      margin-top: calc(@wizard-pf-steps-indicator-margin-top-bottom + 10px);
```

Comment:
IMO, the steps area should be allowed to expand if the steps begin to stack.  No amount of extra margin is going to accommodate and indeterminate number of steps.